Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -64,6 +65,11 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager, El
         leftTicksPerFiveFeet = propManager.createPersistentProperty("leftDriveTicksPer5Feet", 0);
         rightTicksPerFiveFeet = propManager.createPersistentProperty("rightDriveTicksPer5Feet", 0);
 
+        leftMaster.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 0);
```

Comment:
Did this turn out to work?

Reply:
It seems so - it may have been coincidence (they were on a master branch that was a bit behind, where there were a lot of exceptions being thrown on the live robot), but it tracked ticks well.